schematic/manifest/commands.py

```diff
@@ -343,14 +339,15 @@ def migrate_manifests(
 )
 @click.pass_obj
 def download_manifest(ctx, dataset_id, new_manifest_name):
-    master_fileview = CONFIG["synapse"]["master_fileview"]
-
     # use Synapse Storage
     store = SynapseStorage()
 
     # download existing file
     manifest_data = store.getDatasetManifest(
-        datasetId=dataset_id, downloadFile=True, newManifestName=new_manifest_name
+        datasetId=dataset_id,
+        downloadFile=True,
+        newManifestName=new_manifest_name,
+        use_temporary_folder=False,
```

Comment:
When running in CLI mode we will not be downloading manifest files to a temporary folder, instead continue to use the defined manifest folder from the config.

Reply:
Added! @linglp

Reply:
Thanks Bryan!